sources/packages/backend/apps/workers/src/controllers/supporting-user/_test_/e2e/supporting-user.controller.createIdentifiableSupportingUsers.e2e-spec.ts

```diff
@@ -0,0 +1,306 @@
+import { ApplicationData, SupportingUserType } from "@sims/sims-db";
+import {
+  createE2EDataSources,
+  createFakeSupportingUser,
+  E2EDataSources,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import {
+  createFakeWorkerJob,
+  MockedZeebeJobResult,
+} from "../../../../../test/utils/worker-job-mock";
+import { createTestingAppModule } from "../../../../../test/helpers";
+import { SupportingUserController } from "../../supporting-user.controller";
+import {
+  CreateIdentifiableSupportingUsersJobInDTO,
+  CreateIdentifiableSupportingUsersJobOutDTO,
+} from "../../supporting-user.dto";
+import { createFakeCreateIdentifiableSupportingUsersPayload } from "./create-identifiable-supporting-user";
+import { ICustomHeaders } from "@camunda8/sdk/dist/zeebe/types";
+import * as faker from "faker";
+import {
+  APPLICATION_NOT_FOUND,
+  SUPPORTING_USER_FULL_NAME_NOT_RESOLVED,
+} from "@sims/services/constants";
+
+describe("SupportingUserController(e2e)-createIdentifiableSupportingUsers", () => {
+  let db: E2EDataSources;
+  let supportingUserController: SupportingUserController;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    db = createE2EDataSources(dataSource);
+    supportingUserController = nestApplication.get(SupportingUserController);
+  });
+
+  it("Should create one parent supporting user and save the associated full name when one parent's information is added to the student application.", async () => {
```

Comment:
Minor, for better understanding the description can say, `Should create supporting user for first provided parent` or `Should create supporting user for the provided parent`

Reply:
Updated